src/Application/Bootstrap.php

```diff
@@ -107,13 +107,20 @@ protected function start()
 
     protected function execute()
     {
+
+      JLoader::register('JUri',__DIR__ . '/../Uri/JUri.php');
+      JLoader::register('JHtmlSidebar',__DIR__ . '/../Uri/JHtmlSidebar.php');
+
+      JLoader::register('JAdminCssMenu',__DIR__ . '/../Uri/JAdminCssMenu.php');
+      JLoader::load('JAdminCssMenu');
```

Comment:
Hi @ifandpromiss 

we use PSR 4 with autoload in the project instead of the register method of Joomla. Besides, Joomla will abandon the current way and use autoloader in the 4.x. Therefore, you do not need to register the class